core/domain/html_validation_service_test.py

```diff
@@ -21,7 +21,6 @@
 import logging
 import os
 
```

Comment:
Hmm, why all these deletions? Also I would expect tests for the above functions to be in this file